services/web-app/lib/github.js

```diff
@@ -310,10 +310,21 @@ const getLibraryAssets = async (params = {}) => {
     // if fetching a specific asset, only return that
     return libraryParams.asset ? getSlug(asset.content) === libraryParams.asset : true
   })
-
   return assets
 }
 
+export const getAssetIssueCount = async (asset) => {
+  const { host, org, repo } = asset.params
+  console.log(asset.prams, 'hi')
+  return (
+    (
+      await getResponse(host, 'GET /search/issues', {
+        q: `${asset.content.name}+repo:${org}/${repo}+is:issue+is:open+in:title`
+      })
+    ).total_count ?? 0
+  )
+}
```

Comment:
- [ ] Remove console.log
- [ ] Add JSDoc comment
- [ ] Wrap getResponse in a try/catch like the other examples
- [ ] If there's a response, safely check for `total_count`
- [ ] Create new typedef for GitHubSearchResponse

I think something like this will work:

```suggestion
/**
 * Gets the GitHub open issue count for an asset using the asset's name searching only issue title
 * @param {import('../typedefs').Asset} asset
 * @returns {number}
 */
export const getAssetIssueCount = async (asset) => {
  const { host, org, repo } = asset.params

  /**
   * @type {import('../typedefs').GitHubSearchResponse}
   */
  let response = {}

  try {
    response = await getResponse(host, 'GET /search/issues', {
      q: `${asset.content.name}+repo:${org}/${repo}+is:issue+is:open+in:title`
    })
  } catch (err) {
    return null
  }

  return response?.total_count ?? 0
}
```